docs/pages/build-reference/infrastructure.mdx

```diff
@@ -29,9 +29,11 @@ When selecting an image for the build you can use the full name provided below o
 
 ## Android build server configurations
 
-Android workers run on Kubernetes in an isolated environment. Every build gets its own container running on a dedicated Kubernetes node:
+Android workers run on Google Cloud Compute Engine machines in an isolated environment. Every build gets its own dedicated VM instance:
```

Comment:
Don't mention Google Cloud. IMO this shouldn't be relevant to our users.

Reply:
I will just say:
`Android workers run on virtual machines in an isolated environment. Every build gets its own dedicated VM instance.`

Reply:
What do you think?

Reply:
Sounds good.